phpBB/memberlist.php

```diff
@@ -1147,6 +1147,20 @@
 					$user_rank_data['img'] .= '<br />';
 				}
 			}
+			// include modules for manage groups link display or not
+			// need to ensure the module is active
```

Comment:
While I can see the intention, I'm not sure if I approve of the general implementation. Including the module functions and instantiating a new instance of p_master() adds overhead that I'm not sure is acceptable.